app/components/common/measureCommitteeList/measureCommitteeList.html

```diff
@@ -1,6 +1,6 @@
 <ul>
   <li ng-repeat="committee in committees">
-    <h4><a ui-sref="appMain.committee.main({committee_id: committee.committee_id})">{{ committee.name }}</a></h4>
+    <h4><a ui-sref="appMain.committee.main({filer_id: committee.filer_id})">{{ committee.name }}</a></h4>
```

Comment:
Not sure where this `committee` comes from, but I think this should be `committee.id`, I don't think there's a `filer_id` on committee.